CI: Fixes to conda environment names

charlesdong1991 opened the pull request:
- [x] closes #23851 
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

charlesdong1991 committed:
change pandas to pandas-dev

charlesdong1991 committed:
change pandas to pandas-dev

datapythonista commented:
@charlesdong1991 are you still working on it? The rest of the changes in the scripts and the travis and azure config described in the issue are missing.

charlesdong1991 commented:
Hi, so sorry that I am having dinner out, but I will carry on working after
back home.
On Thu, 22 Nov 2018 at 20:12, Marc Garcia <notifications@github.com> wrote:

> @charlesdong1991 <https://github.com/charlesdong1991> are you still
> working on it? The rest of the changes in the scripts and the travis and
> azure config described in the issue are missing.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pandas-dev/pandas/pull/23860#issuecomment-441106095>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI1yODPwZQTsFVvOoFA6pY7DtOzcyRdnks5uxvcGgaJpZM4YvksU>
> .
>

datapythonista commented:
Not a problem at all, just said, as it make things more efficient. Enjoy your dinner, and thanks for working on this.

charlesdong1991 committed:
add source activate

charlesdong1991 committed:
add source activate

charlesdong1991 committed:
remove source activate

charlesdong1991 committed:
remove source activate

charlesdong1991 committed:
clean the script

charlesdong1991 committed:
clean the script

charlesdong1991 committed:
debug

charlesdong1991 committed:
debug

charlesdong1991 committed:
debug

charlesdong1991 committed:
debug

datapythonista commented on ci/incremental/setup_conda_environment.cmd:
```diff
@@ -11,11 +11,10 @@ call deactivate
 conda list
 @rem Clean up any left-over from a previous build
-conda remove --all -q -y -n %CONDA_ENV%
+
```
we may need this, but you can replace the `%CONDA_ENV%` by `pandas-dev`

datapythonista commented on ci/incremental/setup_conda_environment.sh:
```diff
@@ -12,15 +12,15 @@ conda list
 # (note workaround for https://github.com/conda/conda/issues/2679:
 #  `conda env remove` issue)
-conda remove --all -q -y -n $CONDA_ENV
+conda remove --all -q -y
```
can you leave the `-n pandas-dev`

datapythonista commented on ci/incremental/setup_conda_environment.sh:
```diff
@@ -12,15 +12,15 @@ conda list
 # Activate first
 set +v
-source activate $CONDA_ENV
+source activate pandas-dev
```
this one can be removed, also the `set +/- v` around it

datapythonista reviewed (changes requested):
Looks good, thanks for working on this @charlesdong1991. Couple of comments, and they should fix the CI which is in red. If after changing them, you see the CI still red, you may want to check what else is needed.

gfyoung committed:
REF/TST: Finish pytest idiom in parser tests (#23863)

Adds more idiom to the following:

* parser/common --> parser/test_common
* parser/quoting --> parser/test_quoting
* parser/usecols --> parser/test_usecols
* parser/python_parser_only -->
parser/test_python_parser_only

Also:

* Finally delete parser/test_parsers!
* Bug in capture_stdout decorator in which
we were forgetting to use compat.wraps.

Builds off of gh-23712.

jbrockmendel committed:
move misplaced modulo test (#23827)

eoveson committed:
TST: Add test cases for GH6173, appending to empty df (#23806)

RomainSa committed:
DOC/TST: doctests leaving extraneous files (#23858)

alimcmaster1 committed:
CLN: Isort core/*.py and core/indexes/ (#23764)

simonjayhawkins committed:
TST: split up pandas/tests/test_multilevel.py (#23797)

charlesdong1991 committed:
add pandas-dev back at two lines

charlesdong1991 committed:
add pandas-dev back at two lines

charlesdong1991 committed:
test again after catching the latest master

charlesdong1991 committed:
Merge branch 'conda_env_name' of github.com:charlesdong1991/pandas into conda_env_name

charlesdong1991 commented:
@datapythonista sorry, this is very wrong... i think i did something very bad when catching up with the latest change, i think i need to re-create a pr... sorry
